### ISSUE TITLE
Optional arguments in chisels

### ISSUE THREAD

**Issue by Erethon** [draios/sysdig#147]
Normally, I would have brought this up on the mailing list, but I think there 
is less traffic there, so I'm posting this here where more people will see it.

I recently discovered sysdig (I really like what I'm seeing so far), so I
might have missed something obvious, if that's the case, sorry in advance.

Currently, chisels can't have optional arguments. There are a lot of cases I
think, were optional args would be really helpful. For example, all the
chisels that use colored output (echo_fds, proc_exec_time, spy_ip, spy_port,
table_generator and topprocs), can't be piped to something else (e.g. `less`)
or to a file, because of all the control characters (`[34m` etc).

To disable colored output, I've edited ansiterminal and spy_port as seen
here [1], but now every chisel that depends on ansiterminal needs an additional
mandatory argument. Optional arguments would be the cleanest solution for this
type of problem I think and they could be easily declared as:

```
{
    name = "name",
    description = "description",
    argtype = "type",
    optional = true
},
```

To sum up, is there support planned for optional arguments in chisels or did I
miss something obvious in the documentation?

[1]: This is just a proof of concept, bffcb7db5df857f940dd45d10ea0cd9ea4234e2d
